### PR TITLE
Role fix

### PR DESCRIPTION
**Pull request by DancingSnow0517**
`roles` is actually a List[int]
add `fetch_roles` method to get the user roles in this guild

### CODE REVIEW

**Comment by DancingSnow0517** [comment]
草，这个commit检查一定得是feat不能feature吗

**Review by TWT233 — CHANGES_REQUESTED** [review]

File: khl/user.py [hunk 1]
```diff
@@ -43,6 +43,20 @@ def __init__(self, **kwargs):
         self._loaded = kwargs.get('_lazy_loaded_', False)
         self.gate = kwargs.get('_gate_', None)
 
+    async def fetch_roles(self, guild_id: str) -> List[Role]:
```
[the remark this answers is not in the document]
Comment (DancingSnow0517) [reply khl/user.py:46]: 拿给guildUser加个属性guild_id?

[the remark this answers is not in the document]
Comment (TWT233) [reply khl/user.py:46]: 我的想法是把 roles 搬到 GuildUser 里，然后把 GuildUser 搬到 guild.py 里，GuildUser可以加一个属性guild/guild_id

**Review by TWT233 — APPROVED** [review]
LGTM

**Pull request merged**